### PR TITLE
Update lv95

### PR DESCRIPTION
**Pull request by Sgachet**

### CODE REVIEW

**Comment by Sgachet** [comment]
Ready !

File: conf/search.conf.part [hunk 1]
```diff
@@ -138,43 +155,6 @@ source src_swissnames3d : src_swisssearch
...
-        , the_geom \
-        , st_y((st_geometryn(the_geom,1))) as x \
-        , st_x((st_geometryn(the_geom,1))) as y \
-        , st_y(st_transform(st_geometryn(the_geom,1),4326)) as lat \
-        , st_x(st_transform(st_geometryn(the_geom,1),4326)) as lon \
-        , 1 as num \
-        , 10 as zoomlevel \
-        , objektart \
-        FROM tlm.swissnames3d_dkm10_blacklist \
-        WHERE objektart = 'Ort' \
```
Comment (gjn) [inline conf/search.conf.part]: Why do we remove the dkm part?

**Comment by gjn** [comment]
The db backends didn't change?

**Comment by loicgasser** [comment]
No not yet , only we do the switch. People are still integrating layers.

File: conf/search.conf.part (hunk 1 above)
Comment (ltclm) [inline conf/search.conf.part]: @loicgasser this has been moved to the materialized view right?

Comment (loicgasser) [inline conf/search.conf.part]: Yep

**Comment by ltclm** [comment]
@gjn please don't deploy the entire stopo_master database next wednesday. i have already created the new materialized view on stopo_prod.
there is an ongoing vec200 integration, the vec200 tables already have new data (release 2017), and they should not yet be deployed to prod.

**Comment by gjn** [comment]
So a trigger to re-create all indices should be enough?

**Comment by ltclm** [comment]
yes, a recreation of the indices is enough.
For each affected database:
```
make deploy-int-config db=xxxx
make deploy-prod-config db=xxxx
```
or after the config deploy you connect to each sphinxsearch instance and do:
```bash
# did not test it
$ indexer --all --rotate --config /etc/sphinxsearch/sphinx.conf
```

**Comment by gjn** [comment]
I'm merging it now and will updatet the indices on dev today.

I'll probably deploy this to production next week.

**Comment by loicgasser** [comment]
We'll have more entries and then we'll remove them when we merge lv95 PR in chsdi?
I think it would be best to make those 2 play together no?

**Comment by gjn** [comment]
Yes, for some time we'll have double entries. I think that's ok.

**Comment by gjn** [comment]
Indices on dev created.

**Pull request merged**